src/main/java/cs2103/v15_1j/jimjim/parser/JJCommandVisitor.java

```diff
@@ -65,6 +65,15 @@ public Command visitAddEventCommonDate(
 		LocalDateTime end = dateTime;
 		return new AddCommand(string, start, end);
     }
+	
+	@Override
+	public Command visitAddEventOneTime(
```

Comment:
Can we make it visistAddEventWithOneTime? Cuz the first time I read it, it sounded like a one-time (as in one occurence) event.